package.json

```diff
@@ -18,10 +18,11 @@
     "copy-core": "shx cp constellation/constellation-core*.* dist/prerequisite",
     "compress-angularsdk": "node compress-with-assets.mjs",
     "copy-extra": "shx cp constellation/lib_asset.json dist/lib_asset.json",
-    "copy-index": "shx cp dist/index.html dist/simpleportal.html && shx cp dist/index.html dist/portal.html && shx cp dist/index.html dist/fullportal.html && shx cp dist/index.html dist/embedded.html && shx cp dist/index.html dist/mashup.html",
+    "copy-index": "shx cp dist/index.html dist/simpleportal.html && shx cp dist/index.html dist/portal.html && shx cp dist/index.html dist/fullportal.html && shx cp dist/index.html dist/embedded.html && shx cp dist/index.html dist/mashup.html && shx cp src/auth.html dist/auth.html",
     "make-mashup-dir": "cd ./dist && mkdir -p prerequisite",
     "build-angularsdk": "rimraf ./dist && npm run build && npm run copy-extra && npm run copy-index && npm run make-mashup-dir && npm run copy-core",
     "prod-build-angularsdk": "rimraf ./dist && npm run prod-build && npm run copy-extra && npm run copy-index && npm run make-mashup-dir && npm run copy-core && npm run compress-angularsdk",
+    "start": "ng serve",
```

Comment:
Please remove this until we decide if/how to deal with this in all 3 of the SDKs and can update the documentation accordingly.